### PR TITLE
Add an action to extract URL from input file

### LINKED ISSUE #160: Make an action to extract URL from input file
We have a [workflow](https://github.com/FerretDB/FerretDB/blob/main/.github/workflows/docs-preview.yml) that builds PR with documentation and deploys it to CloudFlare pages for preview. The way to see that preview URL is currently quite cumbersome. We should improve it.

Write a composite action (similar to extract-docker-tag, but much simpler) that reads "deploy.txt" file, extracts the first encountered URL, and sets the output parameter.

Use that action in the FerretDB workflow to use the deployment URL. See references to this issue.

### PR DESCRIPTION
**Pull request by KrishnaSindhur**
Closes #160.

### CODE REVIEW

File: extract-first-url/main.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	"os"
+	"regexp"
+
+	"github.com/FerretDB/github-actions/internal"
+	"github.com/sethvargo/go-githubactions"
+)
+
+const (
+	regexMatchURL  = `(https?://[^\s]+)`
+	emptyString    = ""
```
Comment (AlekSi) [inline extract-first-url/main.go:29]: I think we could remove that constant :)

**Review by AlekSi — COMMENTED** [review]
Please run linter with `task lint`

**Review by AlekSi — COMMENTED** [review]
Please reformat you code with `task fmt`

**Review by AlekSi — APPROVED** [review]

**Review by noisersup — APPROVED** [review]
LGTM!

**Pull request merged**